Containerfile.marin3r-operator-bundle

```diff
@@ -30,11 +30,16 @@ LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 
 # TODO: cleanup if not needed
-#LABEL com.redhat.component="marin3r-operator-bundle-container" \
-#      name="3scale-mas/marin3r-rhel8-operator-metadata" \
-#      version="${CI_VERSION}" \
-#      summary="Lighweight, CRD based Envoy control plane for Kubernetes" \
-#      description="Lighweight, CRD based Envoy control plane for Kubernetes" \
+LABEL com.redhat.component="marin3r-operator-bundle-container" \
+      name="3scale-mas/marin3r-operator" \
+      version="${CI_VERSION}" \
```

Comment:
this won't work - it's from CPaaS. We must match it it the version we are planning on releasing

Reply:
changed to version="alpha" \. 
Thank you